docs/smart-contracts/marlowe.md

```diff
@@ -92,7 +92,7 @@ There is also an optional _open-role validator_ which enables just-in-time assig
 
 ### Developer Discussions
 
-- [Marlowe on Cardano Forum](https://forum.cardano.org/c/developers/cardano-marlowe/149)
+- [Marlowe on Cardano Forum](https://forum.cardano.org/tag/marlowe)
```

Comment:
A duplicate of a link from the introduction, but I think it works well for it to be in both places.